examples/rbac-remote/README.md

```diff
@@ -0,0 +1,157 @@
+
+## Tools and Projects
+- Kubernetes
+- Feast
+- PostgreSQL Database
+- Keycloak(if oidc)
+
+## Application Environment
+
+This demo contains the following components:
+
+1. Feast Remote Server components (online, offline, registry).
+2. Feast Remote Client RBAC example.
+3. Yaml Configuration and installation related scripts files.
+ 
+![demo.jpg](demo.jpg)
+
+## Setup Instructions
+
+The application works under OpenShift or Kubernetes with the following instructions assume that you are using a OpenShift or Kubernetes cluster.
+
+### Prerequisites
+
+1. Kubernetes Cluster and Kubernetes CLI (kubectl).
+2. Helm: Ensure you have Helm installed for deploying the Feast components. 
+3. Python environment.
+4. Feast CLI latest version.
+
+## 1. Prerequisites Step
+
+ - **Step 1 : Create the Feast project with PostgreSQL.**
+
+    * Install the PostgreSQL on a Kubernetes cluster if you are using OpenShift you can install using [OpenShift Template](https://github.com/RHEcosystemAppEng/feast-workshop-team-share/tree/main/feast_postgres#1-install-postgresql-on-openshift-using-openshift-template)
+    * Port Forward PostgreSQL Database to your local machine. Since we are setting up the Feast project locally using the Feast CLI, so for local connection required port forward PostgreSQL:
+       ``` kubectl port-forward svc/postgresql 5432:5432```
+    *  Create a feature repository/project using cli with PostgreSQL. Please see the instruction for more details [here](https://docs.feast.dev/reference/offline-stores/postgres#getting-started).
+       For this example (local) setup, we create project with name [server](server) using these settings for [feature_store.yaml](server/feature_repo/feature_store.yaml).
+    
+## 2. Authorization Setup
+
+### A. Kubernetes Authorization
+- **Step 1: Create Remote configuration Files**
+    -  Set auth type `kubernetes` in the respective `feature_store` files
+
+         ```yaml
+                auth:
+                   type: kubernetes
+          ``` 
+    - Feature store YAML files for each server, can be created for example like below server.
+
+        **Registry Server:**  [feature_store_registry.yaml](server/k8s/feature_store_registry.yaml)
+        
+        **Offline Server :** [feature_store_offline.yaml](server/k8s/feature_store_offline.yaml)
+        
+        **Online Server :** [feature_store_online.yaml](server/k8s/feature_store_online.yaml)
+
+- **Step 2: Deploy Server Components**
+  - Run the installation script. The setup script will deploy the server components based on user confirmation, type `k8s` for kubernetes authentication deployment. The script will deploy all the components with the namespace name `feast-dev`.
+
+    ```sh
+    ./install_feast.sh
+    ```
+
+### B. OIDC Authorization
+- **Step 1: Setup the keycloak see the documentation [here](https://www.keycloak.org/getting-started/getting-started-kube)**
```

Comment:
I think this is not a reproducible example, if we just link the K8s installation page of Keycloak.
IMO we should provide a ready-to-use setup. Without it, users might be discouraged from trying it out due to the potential complexity and required expertise.

Reply:
updated the read me to create the realm, client, roles users and mapping the roles.